.github/workflows/build.yml

```diff
@@ -156,29 +158,81 @@ jobs:
 
     # ntp-client
```

Comment:
I would suggest this partitioning:

```

These are all subset='network':

    - name: Win32-network [build]
    - name: ntp-client [build]
    - name: io-sim-classes [build]
    - name: io-sim [build]
    - name: typed-protocols [build]
    - name: typed-protocols-examples [build]
    - name: network-mux [build]
    - name: ouroboros-network-framework [build]
    - name: ouroboros-network [build]

These are subset='consensus':

    - name: ouroboros-consensus [build]
    - name: ouroboros-consensus-mock [build]
    - name: ouroboros-consensus-test [build]
    - name: ouroboros-consensus-mock-test [build]
    - name: ouroboros-consensus-byron [build]
    - name: ouroboros-consensus-byronspec [build]
    - name: ouroboros-consensus-byron-test [build]
    - name: ouroboros-consensus-shelley [build]
    - name: ouroboros-consensus-shelley-test [build]
    - name: ouroboros-consensus-cardano [build]
    - name: ouroboros-consensus-cardano-test [build]

```

Reply:
After reading this comment, I realised I mis-classified some of the packages under `consensus` rather than `network` which is now fixed.  Is this what you meant?

Reply:
Yep, the `if: matrix.subset ==` guards all look as expected to me now. 👍